### PR TITLE
Clarified aborting examples and readme

### LINKED ISSUE #135: AbortableAsyncIterator type export
Hi,
It seems there is no type export for `AbortableAsyncIterator` class.

When I import:
```ts
import type { AbortableAsyncIterator } from "ollama";
```

It gives me a type error:
```
Module '"ollama"' has no exported member 'AbortableAsyncIterator'.
```

### PR DESCRIPTION
**Pull request by ParthSareen**
- Updated example to show the full impact of `ollama.abort()`
- Removed example using `AbortableAsyncIterator` - the onus of managing a 1:1 mapping between stream to client is on the user if a client is not passed to the user - README specifies best practice under the abort section
- Closes #135

### CODE REVIEW

File: examples/abort/abort-all-requests.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import ollama from 'ollama'
+
+// Set a timeout to abort all requests after 1 second
+setTimeout(() => {
+  console.log('\nAborting all requests...\n')
+  ollama.abort()
+}, 1000) // 1000 milliseconds = 1 second
```
Comment (BruceMacD) [inline examples/abort/abort-all-requests.ts:7]:
```suggestion
// Set a timeout to abort all requests after 5 seconds
setTimeout(() => {
  console.log('\nAborting all requests...\n')
  ollama.abort()
}, 5000) // 5000 milliseconds = 5 seconds
```

This was 1 second previously, but I'd like to give it a bit more time to load the model into memory.

File: examples/abort/abort-single-request.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+// Create multiple ollama clients
+const client1 = new Ollama()
+const client2 = new Ollama()
+
+// Set a timeout to abort just the first request after 1 second
+setTimeout(() => {
+  console.log('\nAborting dragons story...\n')
+  // abort the first client
+  client1.abort()
+}, 1000) // 1000 milliseconds = 1 second
```
Comment (BruceMacD) [inline examples/abort/abort-single-request.ts:12]:
```suggestion
// Set a timeout to abort just the first request after 5 seconds
setTimeout(() => {
  console.log('\nAborting dragons story...\n')
  // abort the first client
  client1.abort()
}, 5000) // 5000 milliseconds = 5 seconds
```

**Review by BruceMacD — COMMENTED** [review]
Minor comments about giving the models time to load into memory, but this seems like a good approach to this problem.

File: examples/abort/abort-all-requests.ts (hunk 1 above)
Comment (ParthSareen) [reply examples/abort/abort-all-requests.ts:7]: That's a good idea!

**Review by BruceMacD — APPROVED** [review]
Thanks!

**Comment by ParthSareen** [comment]
We've given it some thought and think it is best if stream management is done by the user. We'll move forward with 1 client per stream at the moment. Examples are updated to reflect this sentiment and add clarity.

**Pull request merged**